inventory

```diff
@@ -1,3 +1,3 @@
 
-local ansible_host=127.0.0.1 ansible_connection=local
+local ansible_host=127.0.0.1 ansible_connection=local ansible_python_interpreter=/usr/local/bin/python
```

Comment:
I will remove it from here. This configuration is specific to our image so it should be set on calling ansible-playbook inside container, not here.